draft-ietf-rats-corim.md

```diff
@@ -267,6 +267,8 @@ For more detail, see {{sec-corim-profile-types}}.
 A CoRIM can be signed ({{sec-corim-signed}}) using COSE Sign1 to provide end-to-end security to the CoRIM contents.
 When CoRIM is signed, the protected header carries further identifying information about the CoRIM signer.
 Alternatively, CoRIM can be encoded as a #6.501 CBOR-tagged payload ({{sec-corim-map}}) and transported over a secure channel.
+If a CoRIM is contained within a larger signed document or message, there MUST be a single signer assigned to the scope of the CoRIM as part of the document's specification or protocol conveying the message, respectively.
```

Comment:
I would phrase the statement something like this.

`A CoRIM can be used without signature protection (please see unsigned-corim-map),

For example, an application may use TLS to transmit an unsigned CORIM and specify that the signer
is the public key in the Certificate Message.

Alternatively an unsigned CoRIM can be a part of a RATS Message which itself is signed (example a signed CMW).
In such cases the authority of CoRIM is delegated to the signer encapsulating the CoRIM.`